src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/matchxpath/Example2.java

```diff
@@ -0,0 +1,24 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MatchXpath">
+      <property name="query"
+       value="//CTOR_DEF[count(./PARAMETERS/*) > 0]"/>
```

Comment:
Add more space to the left of `value=`
It should be easy to read xml to see that it is just another attribute under `property`

Reply:
Done.